main.go

```diff
@@ -87,6 +153,8 @@ func (a *args) getopt(args []string) {
 			a.W = arg[2:]
 		case arg == "-c":
 			a.c = true
+		case arg == "-x":
```

Comment:
Please find a flag name that is not used by gcc, it uses -x for settin the language of the next argument. That's why I started to use "silly" flag names like '-99lib'. The long term goal is to be able to do somehting like `CC=99c ./configure && make`, so we must avoid using existing gcc flags.